kwok/README.md

```diff
@@ -47,12 +47,54 @@ spec:
 EOF
 ```
 
+## Specifying Instance Types
+
+By default, Karpenter will create a hypothetical set of instance types that it uses for node provisioning.  You can
+specify a custom set of instance types by providing a JSON file with the list of supported instance options.  Specify
+the location of this file using the `KWOK_INSTANCE_TYPES_FILE` environment variable.
+
+The format for this file is as follows:
+
+```
+[
+    {
+        "name": "<instance type name>",
+        "offerings": {
+            // Karpenter instance type offerings,
+        },
+        "architecture": "(arm64|amd64)",
+        "operatingSystems": {
+            "linux": {}
+        },
+        "resources": {
+            "cpu": "<cpu count>",
+            "memory": "<memory amount>",
+            // ...
+        }
+    },
+    // ...
+]
+```
+
+The offerings struct has the following format:
+
+```
+{
+    "capacityType": "(on-demand|spot)",
+    "zone": "<zone or region name>",
+    "price": <instance type price>,
+}
+```
+
+There is an example instance types file in [examples/instance\_types.json](examples/instance_types.json).
```

Comment:
There is another (small, easily readable) file in kwok/examples/instance_types.json.  Since the link is relative to the README location I don't think the kwok prefix is needed?